packages/web-awesome/src/components/TestResult/TestResultTeardown/index.tsx

```diff
@@ -17,18 +18,27 @@ const typeMap = {
 
 export type TestResultTeardownProps = {
   teardown: AllureAwesomeTestResult["teardown"];
+  id?: string;
```

Comment:
Why `id` is optional? We use it on line `25` without fallback value, so it's `undefined-teardown` by default. As far, as I see, it can't be used without `id`, so let's make the parameter required.